main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        players_data = json.load(f)
+
+    for player_name, player_data in players_data.items():
+        race_data = player_data["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data.get("description", "")}
+        )
+
+        for skill_data in race_data.get("skills", []):
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                bonus=skill_data["bonus"],
+                race=race
+            )
+
+        guild_data = player_data["guild"]
+        if guild_data:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                defaults={"description": guild_data.get("description")}
```

Comment:
To avoid a KeyError, consider using `guild_data.get('description', '')` to provide a default empty string if 'description' is not present in the JSON data.